js/src/elements/form-behavior.ts

```diff
@@ -675,10 +683,16 @@ function setBusySubmitButton(event: Event, form: HTMLFormElement, isLoading: boo
 }
 
 export function getLabel(input: HTMLValueElement): string {
-	const label =
-		input.labels && input.labels.length
-			? input.labels[0].textContent
-			: input.getAttribute('aria-label');
+	let label: string | null = null;
+
+	if (input.type === 'radio') {
```

Comment:
Is there a way we can get this to work without the introduction of a new `data-` specifically for `radio`?

Reply:
After a little more thinking, it'd be idea if we could consider introducing the concept of radio groups or even checkbox groups. Something that says conceptual one input in this group is required. `loc-input-group-required="{inputGroupId} is required."`